app/src/main/java/ch/benediktkoeppel/code/droidplane/HorizontalMindmapView.java

```diff
@@ -604,7 +616,50 @@ private int getVisiblePixelOfLeftmostColumn() {
 
         return numVisiblePixelsOnColumn;
     }
+    
+    /** Show a dialog to input the search string and fires the search. */
+    void startSearch() {
```

Comment:
Would be nice to have a label or description of the input box here, or a placeholder text. Otherwise it's just a popup with an empty input box. Or see my suggestion for how the search behaves on Firefox Mobile, maybe we can adopt it like that?

Reply:
I wanted to do it like Chrome or Firefox mobile, but I realized I'd need to learn too much of Android, maybe in a future release.
Agree to add a label though.

Reply:
Here is how they do it in Brave, that's perfect I think:
![Screenshot_20240628_202957_Brave](https://github.com/bekoeppel/droidplane/assets/50194/f3d5cd61-59c8-41fa-be50-de36d0df9f1e)

Reply:
Right that's perfect: 
* Searched string ✅
* Current occurrence number ✅ / Total of search results ✅
* Reach Previous ✅ / Next occurrence ✅
* Reset search or close search bar ✅

According to me ... nothing is missing 👏.